exercises/armstrong-numbers/description.md

```diff
@@ -0,0 +1,10 @@
+An [Armstrong number](https://en.wikipedia.org/wiki/Narcissistic_number) is a number that is the sum of its own digits each raised to the power of the number of digits.
+It also depends on the base of the number being used (for example base = 10 for the decimal system or base = 2 for the binary system).
+
+For example:<sup></sup>
```

Comment:
While HTML is valid Markdown, we're also aiming to keep these files readable without rendering. As such, let's get rid of empty tags.

Reply:
I removed the empty tag. Do you want no html at all in the description-Markdown then?